views/pages/addTask.ejs

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <%- include('../partials/head'); %>
+</head>
+<body class="container">
+
+<header>
+  <%- include('../partials/header'); %>
+</header>
+
+<main>
+  <p><%= message %></p>
+<form action="/tasks" method="post">
+  <label for="name">Name:</label><br>
+  <input name="name"><br>
+  <label for="complete">Completed: </label>
+  <input type="checkbox" name="complete" value="true"><br><br>
+  <button type="submit">Add</button>
```

Comment:
nit: can we make this add button as pretty as the "add a task" button at the `/tasks` route by using some of the same CSS classes?

Reply:
ya sure. Done! its just the bootstrap Css class for button

Reply:
class="btn btn-primary"